docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ResizeImages.cs

```diff
@@ -37,7 +37,7 @@ public static void Example()
             var imagesFolder = Path.GetDirectoryName(imagesDataFile);
             // Image loading pipeline. 
             var pipeline = mlContext.Transforms.LoadImages(imagesFolder, ("ImageReal", "ImagePath"))
-                        .Append(mlContext.Transforms.ResizeImages("ImageReal", imageWidth: 100, imageHeight: 100));
```

Comment:
>imageWidth [](start = 79, length = 10)

I'd vote to keep them at ImageWidth and ImageWeight. Doesn't feel like output adds information.